modules/portmaster/src/app/shared/config/generic-setting/generic-setting.ts

```diff
@@ -414,6 +415,10 @@ export class GenericSettingComponent<S extends BaseSetting<any, any>> implements
     return this.expertiseService.change;
   }
 
+  get currentExpertiseLevel() {
+    return this.expertiseService.currentLevel;
+  }
```

Comment:
Better use the observable (`this.expertiseService.change`) and a `| async` pipe in the template in order to make sure change detection runs properly. 
The component is configured as `ChangeDetectionStrategy.OnPush` so changes the the expertise level will not trigger a change detection round in the component unless the observable is used.

Reply:
That's nice! Fixed!